internal/node/node_require_patch.js

```diff
@@ -0,0 +1,503 @@
+/**
```

Comment:
this file appears in GH code review as completely new. Can you see if there's some git operation you can do to make the delta look smaller?

Reply:
done. If I also rename `node_loader.js` => `loader.js` than `loader.js` will be the new file (nice and short) and git will treat the rename to be `node_loader.js` => `require_patch.js` making the diff better.